docs/examples/driver_examples/QCoDeS example with Galil DMC4133 Controller.ipynb

```diff
@@ -0,0 +1,895 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bbba7248",
+   "metadata": {},
+   "source": [
+    "# QCoDeS example with Galil DMC4133 Controller"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86f9a772",
+   "metadata": {},
+   "source": [
+    "Purpose of this notebook is to demonstrate how Galil DMC4133 Controller driver along with Arm class can be used for running measurements while controlling the arm head simulteneously. Before begining,\n",
```

Comment:
@Akshita07 i am missing the description of the "lower level" control capabilities in this notebook, assuming that the arm class is not used. would it be fair to add that description? the reason i think it's very beneficial is that the "lower level" capabilities that the arm class hides could also be very useful. imagine when a user wants to control galil motors with qcodes but the arm class does not match the use-case, hence the arm class can't be used, but all the rest of the implemented features can, right? if so, then let's also present them in the notebook.

Reply:
Yes, that is correct. All the low level features can be used without arm class also. Ok, I will add a section for it in the beginning of the notebook.